tests/test_extension.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+import os
+import random
+import string
+from contextlib import contextmanager
+
+import pytest
+from kafka import SimpleClient, SimpleConsumer

```

Comment:
AFAIK SimpleClient and SimpleConsumer are deprecated (see http://kafka-python.readthedocs.io/en/master/changelog.html#feb-15-2016). Could you please switch to KafkaConsumer / KafkaProducer?